test/unit_tests/braket/circuits/test_circuit.py

```diff
@@ -447,6 +448,326 @@ def test_ir_non_empty_instructions_result_types_basis_rotation_instructions():
     assert circ.to_ir() == expected
 
 
+@pytest.mark.xfail(raises=ValueError)
```

Comment:
Could you also add tests for circuits with noise operations?

Reply:
Done in https://github.com/aws/amazon-braket-sdk-python/pull/253/commits/341bcc49f77298b0e4f9dcd1440c07c9b0d2ff0f